veracruz-utils/src/platform/vm.rs

```diff
@@ -57,24 +57,15 @@ pub enum RuntimeManagerMessage {
     /// Vec<Vec<u8>> - The certificate chain for the enclave
     Initialize(String, Vec<Vec<u8>>),
     #[cfg(feature = "linux")]
-    /// A request to initialize the Runtime Manager enclave with the provided
-    /// policy (for Linux).  Parameters (in order):
-    ///
-    /// - The JSON policy in String format,
-    /// - The bytes of the challenge,
-    /// - The challenge ID.
-    Initialize(String, Vec<u8>, i32),
+    /// A request to initialize the Runtime Manager enclave.
+    Initialize,
     #[cfg(feature = "linux")]
```

Comment:
I was thinking that by changing Linux to perform `Initialize` without a policy (matching with Nitro), we could make `nitro` and `linux` match better (removing the `#[cfg(feature =...)]`s that are interspersed throughout this file.

What prevented that for this change?

Reply:
I think that can be changed, thinking more...

Reply:
I've pushed changes to move the Linux attestation process closer to Nitro's.